blocitoff/app/controllers/items_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ItemsController < ApplicationController
+  def new
+    @user = current_user
+    @item = Item.new
+  end
+
+  def create
+    @user = current_user
+    @item = @user.items.build(item_params)
+    
+    if @item.save
+      flash[:notice] = "Item was saved successfully."
+      redirect_to [@user]
```

Comment:
@jboltz88 Very minor - the array is optional when passing a single object to `redirect_to`.